bodhi/client/__init__.py

```diff
@@ -508,11 +522,21 @@ def query_buildroot_overrides(url, user=None, **kwargs):
     Args:
         user (unicode): If supplied, overrides for this user will be queried.
         staging (bool): Whether to use the staging server or not.
+        mine (bool): Whether to use the --mine flag was given.
         url (unicode): The URL of a Bodhi server to create the update on. Ignored if staging is
                        True.
         kwargs (dict): Other keyword arguments passed to us by click.
     """
     client = bindings.BodhiClient(base_url=url, staging=kwargs['staging'])
+    if mine:
+        if user:
+            click.echo('Warning: --mine and --user options both set. --user will be ignored.\n',
+                       err=True)
```

Comment:
So one problem with this warning is that it will be displayed if the user uses ```USERNAME``` - that effectively comes in as ```--user```, so this causes the warning:

```
USERNAME="bowlofeggs" bodhi overrides query --mine
```

I wonder if it would be best to just not do these warnings? I struggle to think of a better way to handle it without falsely warning the user.